src/steps/groups.ts

```diff
@@ -59,14 +57,9 @@ export async function fetchGroups(
       if (workspaceEntity.slug) {
         const slug: string = <string>workspaceEntity.slug;
         await apiClient.iterateGroups(slug, async (group) => {
-          const groupEntity = (await jobState.addEntity(
-            createIntegrationEntity({
-              entityData: {
-                source: group,
-                assign: createGroupEntity(group),
-              },
-            }),
-          )) as BitbucketGroupEntity;
+          const groupEntity = createGroupEntity(group);
+          await jobState.addEntity(groupEntity); // TODO: Make addEntity return the type of the argument
```

Comment:
Whats the purpose of this TODO? Does it mean something like this?
```suggestion
          const { _type: groupEntityType } = await jobState.addEntity(groupEntity);
```

Reply:
https://github.com/JupiterOne/sdk/issues/497, will link to it in comment.